framework/simd.conf

```diff
@@ -81,34 +81,48 @@ avx512fp16		Leaf07_00EDX	    23	avx512f,f16c	# AVX512 16-bit Floating Point
 amx-tile		Leaf07_00EDX	    24		# Advanced Matrix Extensions Tile support
 amx-int8		Leaf07_00EDX	    25	amx-tile	# AMX Tile multiplication for Int8
 sha512			Leaf07_01EAX	    0	avx	# SHA-512 instructions
-#sm3			Leaf07_01EAX	    1	avx	# SM3 instructions
-#sm4			Leaf07_01EAX	    2	avx	# SM4 instructions
+sm3			Leaf07_01EAX	    1	avx	# SM3 instructions
+sm4			Leaf07_01EAX	    2	avx	# SM4 instructions
 raoint			Leaf07_01EAX	    3		# Remote Atomic Operations, Integer
 avxvnni			Leaf07_01EAX	    4	avx	# AVX (VEX-encoded) versions of the Vector Neural Network Instructions
 avx512bf16		Leaf07_01EAX	    5	avx512f	# AVX512 Brain Float16
+lass			Leaf07_01EAX	    6		# Linear Address Space Seperation
 cmpccxadd		Leaf07_01EAX	    7		# CMPccXADD instructions
 #zlmovsb		Leaf07_01EAX	    10		# Zero-length MOVSB
 #fsrs			Leaf07_01EAX	    11		# Fast Short (REP?) STOSB
 #fsrc			Leaf07_01EAX	    12		# Fast Short (REP?) CMPSB, SCASB
-#fred			Leaf07_01EAX	    17		# Flexible Return and Event Delivery
-#lkgs			Leaf07_01EAX	    18		# Load into Kernel GS
+fred			Leaf07_01EAX	    17		# Flexible Return and Event Delivery
+lkgs			Leaf07_01EAX	    18		# Load into Kernel GS
 amx-fp16		Leaf07_01EAX	    21	amx-tile	# AMX Tile multiplication in FP16
 avxifma			Leaf07_01EAX	    23	avx	# AVX-IFMA instructions
-#lam			Leaf07_01EAX	    26		# Linear Address Masking
+lam			Leaf07_01EAX	    26		# Linear Address Masking
+msrlist			Leaf07_01EAX	    27		# RDMSRLIST, WRMSRLIST, IA32_BARRIER_MSR
+movrs			Leaf13_01EAX	    31		# MOV instructions with Read-Shared hint
 #pbndkb			Leaf07_01EBX	    1		# Total Storage Encryption (TSE) instruction PBNDKB
-avxvnniint8		Leaf07_01EDX	    4	avx	# AVX Vector Neural Network Instructions, Int8
+avxvnniint8		Leaf07_01EDX	    4	avxvnni	# AVX Vector Neural Network Instructions, Int8
 avxneconvert		Leaf07_01EDX	    5	avx	# AVX Non-Exception BF16/FP16/FP32 Conversion instructions
-#amx-complex		Leaf07_01EDX	    8	amx-tile	# AMX Complex Matrix multiplication
-avxvnniint16		Leaf07_01EDX	    10	avx	# AVX Vector Neural Network Instruction, Int16
+amx-complex		Leaf07_01EDX	    8	amx-tile	# AMX Tile multiplication for complex matrices
+avxvnniint16		Leaf07_01EDX	    10	avxvnni	# AVX Vector Neural Network Instruction, Int16
 #prefetchiti		Leaf07_01EDX	    14		# PREFETCHIT0/1 instructions
+user_msr		Leaf07_01EDX	    15		# Userspace MSR support (URDMSR and UWRMSR)
 #avx10			Leaf07_01EDX	    19	avx	# AVX10 Converged Vector ISA
+uiret			Leaf07_01EDX	    17		# User-Interrupt Return
```

Comment:
In theory, we could use UIRET because it can be run from any CPL. But I don't think Linux has enabled support for it, so leave it commented out.